main.go

```diff
@@ -44,6 +45,8 @@ type config struct {
 	VaultTTL       time.Duration `envconfig:"VAULT_TOKEN_TTL"`
 	VaultAuthType  string        `envconfig:"VAULT_AUTH_TYPE"`
 	VaultAuthMount string        `envconfig:"VAULT_AUTH_MOUNT_POINT"`
+	VaultRoleID    string	     `envconfig:"VAULT_ROLE_ID"`
+	VaultSecretID  string        `envconfig:"VAULT_SECRET_ID"`
```

Comment:
I recommend using `VAULT_APPROLE_SECRET` and `VAULT_APPROLE_ID` so that the purpose of these variables is more explicit.